### ISSUE TITLE
[Bug Report] date-picker default-time props not work

### ISSUE THREAD

**Issue by TothingWay** [element-plus/element-plus#1304]
### Element Plus version
1.0.1-beta.27

### OS/Browsers version
MacOS10.15.7 / 87.0.4280.141

### Vue version
3.0.5

### Reproduction Link
https://codepen.io/tothingway/pen/QWKYgwp?editors=1111

### Steps to reproduce
1. use default-time prop in el-date-picker
2. default-time is not work
3. warning appears in console when I use other custom props, like <el-date-picker :abc="myProp" />

### What is Expected?
1. default-time work
2. warning for custom props disappears

### What is actually happening?
1. default-time is not work
2. warning appears in console when I use other custom props, like <el-date-picker :abc="myProp" />

**Comment by zazzaz** [comment]
I guess maybe you want to set the default value of DatePicker, in this way, you should use `default-value` according to the doc.

**Comment by zazzaz** [comment]
> warning appears in console when I use other custom props, like 

Can you provide a screenshot, I didn't see it on the demo link.

**Comment by TothingWay** [comment]
@zazzaz No, I want to set the time to the last second of the day after selecting the time,but now after selecting the date, the time will reset to 00:00:00. I want to set the time to 23:59:59 after selecting the date

**Comment by TothingWay** [comment]
@zazzaz I updated the demo link, you can see the warning in the console

**Comment by zazzaz** [comment]
Can I ask, what's this filterable used for?

**Comment by TothingWay** [comment]
I render the component dynamically with Component is api and set some components props by default. filterable is used when the component is el-select, it should not be valid for the date component and should not be warned

**Comment by TothingWay** [comment]
@zazzaz can u fix default-time props ASAP?

**Comment by zazzaz** [comment]
As you could see on the doc, default-time is only used in date-range-picker

**Comment by TothingWay** [comment]
What I see is non-range: Date, range: Date[]

**Comment by zazzaz** [comment]
Correct, my mistake. 

You can use the following code to set the default time to `01:01:01`
```
<el-date-picker
    v-model="value"
    :default-time="new Date(2000,1,1,1,1,1)"
  />
```

**Comment by TothingWay** [comment]
It doesn't work. You can watch this [demo](https://codesandbox.io/s/element-plus-spa-forked-dbfsv)

**Comment by zazzaz** [comment]
I can confirm this is a bug in date-time-picker, not date-picker. will fix soon

**Comment by TothingWay** [comment]
ok,thanks

**Comment by yirandian** [comment]
Has the problem been solved

**Comment by yirandian** [comment]
It still doesn't seem to work

**Comment by Alanscut** [comment]
it has been fixed in some version.  `default-time` could work: https://codepen.io/Alan_scut/pen/PojKBOE, but can't change the time, same as #3244, and could be fixed by #3348

**Comment by Alanscut** [comment]
fixed by #4228
